nmap/tests/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -euo pipefail
+
+source bin/ci/test_helpers.sh
+
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
+
+TEST_PKG_IDENT="${1}"
+export TEST_PKG_IDENT
+hab pkg install core/bats --binlink
+hab pkg install core/curl --binlink
```

Comment:
I think we can remove all pkg installs except bats.

Reply:
Hi @Defilan.  You can remove all of the following; my tests passed after removing these:

```diff
--- a/nmap/tests/test.sh
+++ b/nmap/tests/test.sh
@@ -11,13 +11,6 @@ fi
 TEST_PKG_IDENT="${1}"
 export TEST_PKG_IDENT
 hab pkg install core/bats --binlink
-hab pkg install core/curl --binlink
-hab pkg install core/jq-static --binlink
-hab pkg install core/busybox-static
-hab pkg binlink core/busybox-static ps
-hab pkg binlink core/busybox-static netstat
-hab pkg binlink core/busybox-static wc
-hab pkg binlink core/busybox-static uniq
 hab pkg install "${TEST_PKG_IDENT}"

 bats "$(dirname "${0}")/test.bats"
```

Thanks for updating core/nmap.

Reply:
Good catch. I just updated the code. Thanks @predominant !